amnestywebsite/humanity-theme
# Adds the 66/33 text with video block pattern

Ref: https://github.com/amnestywebsite/humanity-theme/issues/174

Adds the 66/33 text with video block pattern

**Steps to test**:
1. Edit a post
2. Open the block inserter and check the new pattern appears under the patterns tab
3. Insert the pattern and check it matches the example in the ticket

Example: http://bigbite.im/i/1Gurvg
